README.md

```diff
@@ -0,0 +1,15 @@
+# Ensembl Web Resolver
+Resolver resolves and redirects external urls to Ensembl ones
```

Comment:
> redirects external urls to Ensembl ones

I don't think they are external urls though. `resolver.ensembl.org/id/ENSG00000139618` isn't an external url; it is resolver's url.

Reply:
I made this update.